assets/js/post_office_search.tsx

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
 import { createRoot } from 'react-dom/client';
 // @ts-ignore
```

Comment:
I don't think it's going to be able to resolve what it's looking for in that file, since the components that are being imported here are not part of that file. I don't know if it's as easy as just re-exporting things in that file to make them "visible", but I'd think it's probably a better option to generate that file with the build process rather than trying to maintain it by hand.